### PR TITLE
For hwAddress replace dashes with colons. Rm isNoise flag

### PR DESCRIPTION
**Pull request by travisgoldie**

### CODE REVIEW

**Comment by thorste** [comment]
Looks ok to me.  We should retest after this change.

File: python/SignaliQ/model/CloudProviderEvent.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def __init__(self, description, environment_id,
             "cloudVMs": [],
         }
         self.internalType = internal_type
-        self.isNoise = True
+        # self.isNoise = False ## Do not specify the isNoise flag
```
Comment (clemep-sios) [inline python/SignaliQ/model/CloudProviderEvent.py:57]: you need isNoise = true for the third part event correlation to work (unless we're going to change backend code) -- just happened to be looking at that code today

Comment (thorste) [reply python/SignaliQ/model/CloudProviderEvent.py:57]: There should now be a check for null in CloudProviderEventCollectionTask (around line 160) that sets the value to true if not specified.

It's still ok to leave it in also.

Comment (clemep-sios) [reply python/SignaliQ/model/CloudProviderEvent.py:57]: ok, yeah, looks like isNoise should get set in all cases (although with the unmapped possibility in there, it's not straightforward to see that that is the case)

either way, being explicit here doesn't hurt...and it makes sure we're not depending on some undocumented internal behavior (which could easily break or change...just saying)

**Comment by srazin** [comment]
The idea was to "hide" that property since it can overwrite the behaviour
and only make it available to us when you know what's going on. For that
reason the idea was to remove that option.

**Review by srazin — APPROVED** [review]
looks good

**Pull request merged**